modules/social_features/social_topic/social_topic.install

```diff
@@ -501,7 +501,7 @@ function social_topic_update_11401(): string {
 /**
  * Remove deprecated permission "override topic revision log entry".
  */
-function social_topic_update_111101(): void {
```

Comment:
This change should land on 11.x as it it BC. Hook update number should not be changed.